cmd/minishift/cmd/start_preflight.go

```diff
@@ -30,6 +30,7 @@ import (
 	configCmd "github.com/minishift/minishift/cmd/minishift/cmd/config"
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	validations "github.com/minishift/minishift/pkg/minishift/config"
+	miniConstants "github.com/minishift/minishift/pkg/minishift/constants"
```

Comment:
@anjannath , please avoid creating another alias as we already have alias dedicated for such packages.

I can give you a page explanation why it is like [`To put axe on your own foot.`](https://www.urbandictionary.com/define.php?term=To%20put%20axe%20on%20your%20own%20foot.) 😄  

See https://github.com/minishift/minishift/search?q=minishiftConstants&unscoped_q=minishiftConstants

Reply:
Agree with @budhram. It should be `minishiftConstants`.

Reply:
Agree with @budhram. It should be minishiftConstants.